### PR TITLE
[GH-19267] Spelling build

### LINKED ISSUE #19267: Spelling errors in master branch
#### Summary
There are typos in mattermost-server

#### Steps to reproduce
How can we reproduce the issue (what version are you using?)
https://github.com/jsoref/mattermost-server/commit/4ff548f1235d2410473542bac8dd6262488c3561#commitcomment-62819286

#### Expected behavior

No typos and a ✅ 

#### Observed behavior (that appears unintentional) 
What did you see happen? Please include relevant error messages and/or screenshots.
❌ 
<details><summary>Unrecognized words (140)</summary>

```
activitated
activiy
acually
adjustement
alfter
allways
Ancilary
attachements
attachemtns
Avaliable
brakets
cannonical
categorywhich
cateory
catetories
cellspcing
Chaned
chaneld
Chanenel
Channe
comforms
commmandline
Complicance
constained
coresponding
corrensponding
couldnt't
Couns
cxt
deacivated
deactive
decprecated
depreacting
deriviative
destionation
deteminism
diemsnions
direferen
documentatin
duplcates
elipsis
emai
Emojies
emptively
Enivironment
enteprise
exaple
Exeeded
exepected
Exipry
exmaple
Failied
floag
forign
Fowarded
htp
idenitify
indentify
independant
initializaing
Initializiation
initialzed
innapropriate
invate
ivalid
Managment
markteplace
mattemost
memebership
mermbership
Metadat
metion
multipile
notexist
notifed
notoverriden
occured
ocurrs
oprphaned
orphanded
overrideable
overriden
overrite
overwites
pagnation
panicing
paramater
Patchs
Pemissions
permenent
permisssions
persistant
pontential
Posible
Predictible
prepackged
previoous
privat
promete
qouted
Receipients
recorgnizing
registeration
repoy
requrie
Resctricted
resetted
Respose
Restricions
retranslateed
retreiving
Roundtripper
Rountrippers
schemd
Schme
Searc
serch
shoudn't
should'nt
sigantures
signatrue
singup
somthing
specfified
STMP
Sucess
Suggesitons
Suscription
Sycn
syncronize
syncronizer
Sytem
theoritcally
thumbail
unknwon
updation
Usersname
webhok
Webhoook
wilrdcar
```
</details>

#### Possible fixes

- [ ] #19268
- [ ] strings
- [ ] struct fields and others

### PR DESCRIPTION
**Pull request by jsoref**
#### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

This is limited to tests

#### Ticket Link

Fixes #19267 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+.PHONY: build package run stop run-client run-server run-haserver stop-haserver stop-client stop-server restart restart-server restart-client restart-haserver start-docker clean-dist clean nuke check-style check-client-style check-server-style check-unit-tests test dist prepare-enterprise run-client-tests setup-run-client-tests cleanup-run-client-tests test-client build-linux build-osx build-windows package-prep package-linux package-osx package-windows internal-test-web-client vet run-server-for-web-client-tests diff-config prepackaged-plugins prepackaged-binaries test-server test-server-ee test-server-quick test-server-race start-docker-check migrations-bindata new-migration migration-prereqs
```
Comment (agnivade) [inline Makefile:1]: Can you remove "prepare-enteprise" entirely? We are not using it.

**Review by agnivade — COMMENTED** [review]
Thanks @jsoref. Just a small one.

**Comment by agnivade** [comment]
Also, it seems like we have some conflicts. Mind rebasing with master and pushing it? :)

**Comment by agnivade** [comment]
@jsoref - Just a quick note to avoid force-pushing. We already squash all the commits before merging so it's not an issue to push new commits and helps us identify what changed between commits :)

**Review by agnivade — APPROVED** [review]

**Comment by jsoref** [comment]
Are the tests happy? Sadly I have a circle-ci account and it tends to clobber tests results from PRs...

**Comment by agnivade** [comment]
Force merging because upload-s3 fails for a contributor.

**Pull request merged**